File: android/src/main/kotlin/live/hms/hmssdk_flutter/HMSVideoResolutionExtension.kt
package live.hms.hmssdk_flutter

import live.hms.video.media.settings.HMSVideoResolution

class HMSVideoResolutionExtension {

    companion object{

        fun toDictionary(hmsVideoResolution: HMSVideoResolution?):HashMap<String,Any?>?{

            val args=HashMap<String,Any?>()
            if(hmsVideoResolution == null) return null
            args["height"] = hmsVideoResolution.height/1.0
            args["width"] = hmsVideoResolution.width/1.0

            return args;
        }
    }
}

File: ios/Classes/Models/HMSVideoResolutionExtension.swift
//
//  HMSVideoResolutionExtension.swift
//  hmssdk_flutter
//
//  Created by govind on 29/01/22.
//

import Foundation
import HMSSDK

class HMSVideoResolutionExtension {

        static func toDictionary(_ hmsVideoResolution: HMSVideoResolution) -> [String: Any] {

            var dict = [String: Any]()

            dict["height"] =  hmsVideoResolution.height
            dict["width"] =  hmsVideoResolution.width

            return dict
        }
}
